array_count.js

```diff
@@ -0,0 +1,31 @@
+var arr1=[10,3,5,12,5,3,9,1];
+var len=arr1.length;
+var visited_ele=[];
+var count;
+for(var i=0;i<len;i++)
+{
+  count=1;
+
+  for(var j=i+1;j<len;j++)
+  {
+    if(arr1[i]===arr1[j])
+    {
+      count++;
+      visited_ele[j]=0;
+    }
+    
+  }
+  if(visited_ele[i]!=0)
+  {
+    visited_ele[i]=count;
+  }
+}
+for(var i=0;i<len;i++)
+{
+  if(visited_ele[i]!=0)
+  console.log(arr1[i]+" occured "+visited_ele[i]+" times");
+}
```

Comment:
Can you solve this problem without using a array/object which keeps track of the repetition (in this code the variable is visited_ele).   ie if you were to use C as a programming language how will you solve this ?

Reply:
added a new file **array_count_1.js**, i have tried in a different way, without using array/object to keep the track of repetition.